R/pkg/R/functions.R

```diff
@@ -963,6 +964,7 @@ setMethod("kurtosis",
 #' last(df$c, TRUE)
 #' }
 #' @note last since 1.4.0
+#' @note the function is non-deterministic because its result depends on order of rows.
```

Comment:
Indeterministic, or usually non-deterministic. Shell we match the words while we are here?

Reply:
same here about `@note`
ditto in all other cases

Reply:
@HyukjinKwon I am using `non-deterministic` everywhere. I don't see big difference between `Indeterministic` and `non-deterministic`. Do you believe `Indeterministic` is better fit here?

Reply:
I have seen few indeterministic in other functions... I thought matching it to non-deterministic